docs/modelserving/v1beta1/llm/huggingface/README.md

```diff
@@ -0,0 +1,69 @@
+# Deploy the Llama2 model with Hugging Face LLM Serving Runtime
+The Hugging Face LLM serving runtime implements a runtime that can serve Hugging Face LLM model out of the box. 
+
+In this example, we deploy a Llama2 model from Hugging Face by running an `InferenceService` with [Hugging Face Serving runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver). Based on the performance requirement for large language models, KServe chooses to perform the inference using a more optimized inference engine like [vLLM](https://github.com/vllm-project/vllm) for text generation models.
```

Comment:
I will suggest to add information that a license is needed for this [model](https://huggingface.co/meta-llama).
> Welcome to the official Hugging Face organization for Llama 2 models from Meta! In order to access models here, please visit the [Meta website](https://ai.meta.com/resources/models-and-libraries/llama-downloads) and accept our [license terms](https://ai.meta.com/llama/license/) and [acceptable use policy](https://ai.meta.com/llama/use-policy/) before requesting access to a model. Requests will be processed within 1-2 days.